src/main/java/io/odpf/depot/exception/InvalidMessageException.java

```diff
@@ -0,0 +1,8 @@
+package io.odpf.depot.exception;
+
+public class InvalidMessageException extends RuntimeException {
```

Comment:
Let's make it checked Exception

Reply:
RuntimeException should be thrown when they are not to be declared and caught. A RuntimeException can only be fixed with change in the code and not with configuration.